src/index.ts

```diff
@@ -1,4 +1,6 @@
-import apiClient, { Handlers, Tokens, ApiClientConfig } from "./apiClient"
+import apiClient from "./apiClient"
+// TODO: do we really wanna handle it like this and export the instance as if is a constructor
```

Comment:
TODO

Reply:
I'm ok with this :)

Reply:
maybe this refers to my comment above? https://github.com/carforyou/carforyou-api-client-pkg/pull/175/files#diff-1c1a34a94ec474ad2092388c3650708eR27